.github/workflows/pull_request.yml

```diff
@@ -57,6 +57,53 @@ jobs:
       rendering_docker_image: "${{ env.RENDERING_DOCKER_IMAGE }}"
       dockerfiles_changed: steps.verify-changed-files.outputs.files_changed
 
+  gather-anvil-repos:
+    name: Gather AnVIL Repositories
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout files
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Configure git
+        run: |
+          git config --local user.email "itcrtrainingnetwork@gmail.com"
+          git config --local user.name "jhudsl-robot"
+
+          branch_name='preview-${{ github.event.pull_request.number }}'
+          git fetch --all
+          git checkout $branch_name
+          git merge -s recursive --strategy-option=theirs origin/${{ github.head_ref }} --allow-unrelated-histories
+        shell: bash
+
+      - name: Run AnVIL repo check
+        id: anvil_repos
+        env:
+          GH_PAT: ${{ secrets.GH_PAT }}
+        run: |
+          sudo apt-get install libcurl4-openssl-dev
+          sudo Rscript -e "install.packages(c('optparse', 'curl', 'httr', 'jsonlite', 'dplyr', 'readr'))"
```

Comment:
This takes forever

Reply:
Is that something we could make a custom Docker image to solve?

Reply:
Also, in general (and especially if it's slow), do we want it to run for every PR?  We could make it a manual trigger, or, if there's a way to set GHA to watch a specific file, maybe we could have it run whenever sync.yml is changed (assuming that a change to sync.yml is often because a new "child" repo has been added)?

Reply:
Added the container. Super fast now :)